src/abstract/EpochBasedInflationaryVoteToken.sol

```diff
@@ -131,8 +133,12 @@ abstract contract EpochBasedInflationaryVoteToken is IEpochBasedInflationaryVote
      * @param account_ The address of the account to sync.
      */
     function _sync(address account_) internal virtual {
+        uint240 inflation_ = _getUnrealizedInflation(account_, _clock());
+
+        emit Transfer(address(this), account_, inflation_);
```

Comment:
any benefit to returning early if no unrealized inflation and do not emit useless Transfer event too?

Reply:
EIP20 specifies that Transfer events of 0 value must be emitted.
https://eips.ethereum.org/EIPS/eip-20#transfer-1

Reply:
It actually only says "when tokens are transfered", which implies the `transfer` and `transferFrom` functions, not a "sync". We can skip these event emissions and we'd still emit the regular `Transfer` event anyway.